projects/Website Blocker/website_blocker.py

```diff
@@ -0,0 +1,12 @@
+pathToHosts=r"C:\Windows\System32\drivers\etc\hosts"
+redirect="127.0.0.1"
+websites=["https://www.sislovesme.com/","https://motherless.com/","https://xhamster.com/","https://www.xnxx.com/","https://www.xvideos.com/","https://www.pornhub.com/"]
+
+with open(pathToHosts,'r+') as file:
+content=file.read()
```

Comment:
Make sure this is properly indented. Currently the program produces an indentation Error